### DISCUSSION TITLE
Welcome to geoda Discussions!

### DISCUSSION THREAD

**Discussion by lixun910** [GeoDaCenter/geoda#2289] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edv-geo** [comment]
Hi!
First: Geoda is a great software, thank you for your work!

I'm wondering why the current (1.18 12/10/2020 64bit) Geoda installer provides the "C:\Program Files\GeoDa Software" folder with full permissions for "everyone". Our IT department prohibits the installation for that reason as they tell me that is a security problem. Would it be possible to change that?
Thank you & best regards, edv-geo

**Reply by lixun910** [reply]
@edv-geo Is this related to your question: https://github.com/GeoDaCenter/geoda/issues/2349 ? Thanks!

**Reply by lixun910** [reply]
@dev-geo see #2349 

These suggestions are great! Here is a new build: https://www.dropbox.com/s/vzr4p42k9uy68c3/GeoDa-Windows7%2B-x64-installer.zip?dl=0

* The logger.txt (with other user data) are located to %appdata%.
* The "everyone-fullcontrol" permissions on C:\Program Files\GeoDa software is removed. (only read and execute)
* The %appdata%/GeoDa permissions are set to "users-modified"

I gave it a try following the stackoverflow post, but as you mentioned, it doesn't work if the installation is not run as administrator(c++ redistributable, windows registry). However, I think with the current installer, the administrator ((e.g. in a lab environment) can install GeoDa for all users, which will have their own geoda data under their own %appdata% directory.

**Comment by jkoschinsky** [comment]
Since GeoDa's Windows installer will install the Microsoft Visual C++ 2010 redistributable first, admin permissions are required to install GeoDa on Windows. However, for a lab environment, if the Visual C++ 2010 redistributable is preinstalled, you can distribute GeoDa as a zipped file, see https://geodacenter.github.io/download_windows.html#usbinstall
<https://spatial.uchicago.edu/>

<https://spatial-data-science-conference.com/>
________________________________
